docs/source/content/handling_time_limits.md

```diff
@@ -0,0 +1,49 @@
+# Handling Time Limits
+In using Gym environments with reinforcement learning code, a common problem observed is how time limits are incorrectly handled. The `done` signal received from `env.step` indicates whether an episode has ended. However, this signal does not distinguish between the two different reasons why an episode can end - `termination` and `truncation`. 
+
+### Termination
```

Comment:
I'm still not a big fan of opposing "termination" vs "truncation/timelimit", because a timeout is a termination in practice (even though you treat it differently in infinite horizon problem).
I would more treat as "truncation/termination due to timelimit" vs "other terminations"

Reply:
About the terminology, unless I'm mistaken there is no clear literature on definitions of termination and truncation. But, I've seen states referred to as `terminal states` specifically mean 'special states' after which the episode ends, in the sense that this is built into the MDP. Sutton and Barto for eg. says "Note that the value of the terminal state, if any, is
always zero." and throughout the book uses terminal states in this context. This is not true if we call the last state after a timeout in an infinite-horizon problem also as a terminal state. 

The term `termination` is an extrapolation from this, saying that, "If and only if an environment reaches a terminal state, it is said to have `terminated`". 

And for all other cases, we say it is `truncated`. And we want to make this distinction specifically since it also makes a big difference for bootstrapping.

Reply:
i think they used "absorbing terminal state" too, need to double check